### ISSUE TITLE
Keep getting an error on ArchLinux x86_64 with Intel HD 4000

### ISSUE THREAD

**Issue by ghost** [laanwj/hw2view#1]
Hello,

I tried to run `hw2view` on my machine but nothing shows up and I get the following error outputting in my terminal.

```
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 314, in 'calling callback function'
  File "./visualize.py", line 81, in draw
    glEnable(GL_PRIMITIVE_RESTART)
  File "/usr/lib/python2.7/site-packages/OpenGL/error.py", line 232, in glCheckError
    baseOperation = baseOperation,
OpenGL.error.GLError: GLError(
    err = 1280,
    description = 'invalid enumerant',
    baseOperation = glEnable,
    cArguments = (GL_PRIMITIVE_RESTART,)
)
```

**Comment by laanwj** [comment]
It appears that your OpenGL driver has no support for GL_PRIMITIVE_RESTART. That's fine, actually! But the problem is that hw2view thinks it has.

Can you try to remove these lines : https://github.com/laanwj/hw2view/blob/github/visualize.py#L243-248  
That force-disables the use of primitive restart and you should get an image...

Next point is to find out _why_ it misdetects on your hardware, but let's try this first.

**Comment by laanwj** [comment]
FYI I had the same issue after upgrading ubuntu/PyOpenGL: fixed in 73b08bcdfbd438b3e2869e0f4afa85d3081797d2
